pkg/util/web3/contractid.go

```diff
@@ -12,9 +12,10 @@ type ContractID struct {
 	Blockchain string
 	Network    string
 	Address    string
+	Query      *url.Values
```

Comment:
Use `url.Values` instead of `*url.Values`. `url.Values` is a map so it is a pointer type already. `*url.Values` is a pointer to a pointer, too complicated.